### DISCUSSION TITLE
lxml

### DISCUSSION THREAD

**Discussion by xuying-92** [flet-dev/flet#3284] [Packages] [answered]
https://pypi.org/project/lxml/

**Comment by gryznar** [comment]
+1 lxml is a way faster than bs4. Parsing HTML is complex and fast parser makes real difference

**Comment by FeodorFitsner** [comment, marked as answer]
[`lxml`](https://pypi.flet.dev/lxml) is available in Flet 0.25.0 pre-release and up for both iOS and Android. Give it a try and let us know how that works.
